src/components/Result/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+
+function Result(props) {
+  const user = props.user;
+  const api = `https://api.github.com/users/${user}/repos`;
+  const [repo, setRepo] = useState([]);
+
+  useEffect(() => {
+    const fetchRepos = async () => {
+      try {
+        let { data } = await axios.get(api);
+        setRepo(data);
+      } catch (err) {
+        setError(err.message);
+      }
+    };
+    fetchRepos();
+  }, []);
+
+  function mapRepo() {
+    return repo.map((r) => {
+      return (
+        <div>
```

Comment:
When mapping data to jsx, the outermost element requires the unique `key` attribute - see devtools error for more info